release-pr/pyproject.toml

```diff
@@ -6,7 +6,7 @@ readme = "README.md"
 authors = [
     { name = "JC Grünhage", email = "jc@neon.tech" }
 ]
-requires-python = ">=3.11"
+requires-python = ">=3.10"
```

Comment:
Let's add a comment why we have 3.10 here, so we can bump it in the future (once we move to `ubuntu-24.04`)

Reply:
sgtm